coral/src/app/features/topics/components/search/SearchTopics.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Flexbox,
+  Icon,
+  InputBase,
+  PrimaryButton,
+} from "@aivenio/design-system";
+import { FormEvent, useState } from "react";
+import searchItem from "@aivenio/design-system/dist/module/icons/search";
+
+type SearchTopicsProps = {
+  searchTerm: string;
+  search: (searchTerm: string) => void;
+};
+
+function SearchTopics(props: SearchTopicsProps) {
+  const { search, searchTerm } = props;
+  const [currentSearchTerm, setCurrentSearchTerm] =
```

Comment:
Good point! The prop is already called `searchTerm`, I'm not sure how we can name it better 🤔  That's where that `current` came from (I'm not super creative with naming 🙈 ). Maybe it could just be `value`? 

It would be used like that:

```
<SearchTopics search={searchTopics} value={searchTerm} />
```

I think that's clear?

Reply:
Yes! And also as I commented below, we could rename `search` -> `onChange`